packages/salesforcedx-apex-replay-debugger/src/commands/apexExecutionOverlayResultCommand.ts

```diff
@@ -0,0 +1,162 @@
+  namespace: string;
+}
+
+export interface HeadpDumpExtents {
+  collectionType: string | null;
+  count: number;
+  definition: HeadpDumpCollectionTypeDefinition[];
+  extent: HeadpDumpExtent[];
+  totalSize: number;
+  typeName: string;
+}
+
+export interface HeadpDumpCollectionTypeDefinition {
+  name: string;
+  type: string;
+}
+
+export interface HeadpDumpExtent {
+  address: string;
+  size: number;
+  symbols: string[] | null;
+  value: HeadpDumpExtentValue;
+}
+
+// The Extent value has to be any. The value
+export interface HeadpDumpExtentValue {
+  value?: any;
+  entry?: HeadpDumpExtentValueEntry[];
+}
+
+export interface HeadpDumpExtentValueEntry {
+  keyDisplayValue: string;
+  value: HeadpDumpExtentValue;
+}
+
+export interface HeapDumpApexResult {
+  apexError: string | null;
+  apexExecutionResult: HeapDumpApexExecutionResult | null;
+}
+
+export interface HeapDumpApexExecutionResult {
+  column: number;
+  compileProblem: string | null;
+  compiled: boolean;
+  exceptionMessage: string | null;
+  exceptionStackTrace: string | null;
+  line: number;
+  success: boolean;
+}
+
+// If the queryError is returned then the queryMetadata and queryResult will both be null.
+export interface HeapDumpSOQLResult {
+  queryError: string | null;
+  queryMetadata: HeapDumpSOQLResultQueryMetadata | null;
+  queryResult: HeapDumpSOQLResultQueryResult[] | null;
+}
+
+export interface HeapDumpSOQLResultQueryMetadata {
+  columnMetadata: HeapDumpSOQLResultColumnMetadata[];
+  entityName: string;
+  groupBy: boolean;
+  idSelected: boolean;
+  keyPrefix: string;
+}
+
+export interface HeapDumpSOQLResultColumnMetadata {
+  aggregate: boolean;
+  apexType: string;
+  booleanType: boolean;
+  columnName: string;
+  custom: boolean;
+  displayName: string;
+  foreignKeyName: string | null;
+  insertable: boolean;
+  joinColumns: string[];
+  numberType: boolean;
+  textType: boolean;
+  updatable: boolean;
+}
+
+// A note about the HeapDumpSOQLResultQueryResult. The fields from a SOQL
+// query will be the fields that the user asked for. For instance if Id,
+// Name and AccountNumber were requested returns then the they'd be accessed
+// through the HeapDumpSOQLResult.queryResult['Id'|'Name'|'AccountNumber'].
+// The field name strings would be accessed through the ColumnMetadata returned
+// with the query.
+export interface HeapDumpSOQLResultQueryResult {
+  attributes: Attributes;
+  [fields: string]: any;
+}
+
+export interface Attributes {
+  type: string;
+  url: string;
+}
+
+// There's nothing remarkable in the request body or the query string. This
+// particular command is simply the URL with a query string that is
+// _<CurrentTimeInMilliseconds>
+export class ApexExecutionOverlayResultCommand extends BaseCommand {
+  private readonly commandName = 'ApexExecutionOverlayResult';
+  private readonly heapdumpKey: string;
+
+  public constructor(heapdumpKey: string) {
+    // The query string is '_<milliseconds>'
+    super('_=' + Date.now());
```

Comment:
Is the timestamp necessary? How does it affect the response?

Reply:
It doesn't. I've modified this to pass in undefined for the super.